elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/GsonHolder.java

```diff
@@ -12,18 +12,17 @@
 import io.searchbox.client.AbstractJestClient;
 
 /**
- * Centralizes the configuration of the Gson builder.
+ * Centralizes the configuration of the Gson object.
  *
  * @author Guillaume Smet
  */
-public class GsonBuilderHolder {
+public class GsonHolder {
 
-	private GsonBuilderHolder() {
+	private GsonHolder() {
 	}
 
-	public static final Gson BUILDER = new GsonBuilder()
+	public static final Gson INSTANCE = new GsonBuilder()
```

Comment:
Do you mind calling it GSON instead? I put an instance of the Gson JsonParser here in a following patch.